languages/python/exercises/concept/string-methods/.docs/after.md

```diff
@@ -0,0 +1,26 @@
+## String Methods in Python
```

Comment:
We've recently changed the spec a bit, where the `after.md` document has been replaced with concept-specific documents: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#concept-files It's totally fine to fix this later if you'd like to get this PR merged.

Reply:
Since this (and several other PRs) were being worked on as that re-org took place, we're going to merge them and then do some clean up.